app/models/concerns/decidim/kids/static_page_override.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Kids
+    module StaticPageOverride
+      extend ActiveSupport::Concern
+
+      included do
+        MINORS_DEFAULT_PAGES = %w(minors).freeze
+        Decidim::StaticPage::DEFAULT_PAGES = (Decidim::StaticPage::DEFAULT_PAGES + MINORS_DEFAULT_PAGES).uniq.freeze
```

Comment:
Maybe we should consider adding this slug to the `DEFAULT_PAGES` only if the kids module is activated for the tenant. This would allow admins to remove the `minors` page in case they don't need it

Reply:
I have overridden the permissions class so I check if the page is the "minors" one and the configuration is enabled. The admins can delete this page if the kids' configuration is disabled, and every time the configuration is enabled, the page will be created automatically if it doesn't exist